phpstan.dist.neon

```diff
@@ -0,0 +1,16 @@
+parameters:
+    level: 1
+    paths:
+        - src
+    excludePaths:
+        - src/Behat/Testwork/EventDispatcher/TestworkEventDispatcherSymfonyLegacy.php
+        - src/Behat/Testwork/EventDispatcher/TestworkEventDispatcherSymfony5.php
+    ignoreErrors:
+        -
+            identifier: class.notFound
+            paths:
+                - src/Behat/Testwork/Exception/Stringer/PHPUnitExceptionStringer.php
+        -
+            identifier: constructor.unusedParameter
```

Comment:
This class has a constructor parameter that is not used but it needs to be present because it is expected that all `SimpleArgumentTransformation` classes have it

Reply:
I have added a task for this to https://github.com/Behat/Behat/issues/1519, the issue that we have to track what we want to implement for the next major